StreetDrop/StreetDrop/Data/DataMapping/SearchMusicResponseDTO.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  SearchedMusicList.swift
+//  StreetDrop
+//
+//  Created by 맹선아 on 2023/05/14.
+//
+
+import Foundation
+
+struct SearchMusicResponseDTO: Decodable {
+    let list: [Music]
```

Comment:
@sunny-maeng @joongkyu-park 
Music 구조체에 대한 배열이기에 하기 코드중 하나로 변경 가능할까요?
이에 앞서, 혹시 배열에 대한 프로퍼티명을 통일했으면 하는데, 어떤게 나으신가요?
```suggestion
    let musicList: [Music]
    let musics: [Music]
    let allMusic: [Music]
    let musicArray: [Music]
```
poi는 allPoi로 되어 있더라구요!

Reply:
저는 Array타입이란걸 나타내려고 원래는 뒤에 -s 를 붙이는 편이긴 한데요,,

POI의 복수형으로는 point가 points가 되어서 똑같이 POI 라 생각해서 all을 사용했었습니다 그런데 검색해보니 약자의 복수형으로 약자자체뒤에 s를 붙인다고하네요! POIs->POIs 이건 괜찮으신가요? 

music은 복수형이 따로없지만 -s로 통일해서 musics, pois(코드컨벤션: 약어 첫 단어시작시 소문자시작..) 로 할까요? 

추천해주신 네개중에 더 나아보이는게 있으시면 다른것도 좋습니다!

Reply:
선아님 말씀대로 -s 붙이는 거 동의합니다!

@joongkyu-park 중규님 생각은 어떠신가요?